### ISSUE TITLE
Accessibility issue: Documents must have <title> element to aid in navigation on /az_core/mobile_nav_callback/menu_link_content%3A911b448b-1e04-4a83-bc38-9a13fa16224b

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#69]
## What
An accessibility scan flagged the element `<html>` because documents must have <title> element to aid in navigation. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/document-title?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/01846c1c-38e4-4f8a-ac5b-aa689f68ae07.png)

To fix this, ensure each html document contains a non-empty <title> element.

Specifically:

Fix any of the following:
-   Document does not have a non-empty <title> element

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
